client/python/armada_client/internal/binoculars_client.py

```diff
@@ -0,0 +1,87 @@
+from typing import Optional
+
+import grpc
+
+from armada_client.armada import (
+    binoculars_pb2,
+    binoculars_pb2_grpc,
+)
+
+from armada_client.k8s.io.api.core.v1 import generated_pb2 as core_v1
+
+
+def new_binoculars_client(url: str, disable_ssl: bool = False):
```

Comment:
Should it by default expect a secure connection?

Reply:
The way it's written, by default it will try to use SSL.